### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Websites_Zooming: When browser is resized to 200%, some of the contents and controls disappears and they no longer remains accessible for the user.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#105]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated**: MAS1.4.4

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. set your browser zooming till 200% and verify that all the contents and controls on the page are visible and accessible.

**Actual Result:**
When browser is resized to 200%, following contents and controls disappears and they no longer remains accessible for the user:
"Default App Pool" and "Local Host" link

**Expected Result:**
All the contents and controls of the page should be visible and accessible even after zooming the screen to 200%.

**User Impact:**
Visually Impaired users who uses resize features to enlarge the text on the page, will not be able to access all the controls of the page.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}

**Attachment:**
![bug 105_zooming issue](https://user-images.githubusercontent.com/38521133/44657935-4c41ff80-aa1c-11e8-80ca-1b23671bce6e.png)

**Comment by jhkimnew** [comment]
This is by design.

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
